isv.md

```diff
@@ -24,6 +24,7 @@ Additionally:
 | Containers | [Red Hat](https://partners.amazonaws.com/partners/001E000000Rl0xiIAB) | [OpenShift Container Platform](https://www.redhat.com/en/technologies/cloud-computing/openshift/container-platform) | Release: https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html-single/release_notes/index#ocp-4-10-OCP-on-ARM<br>Announcement: https://cloud.redhat.com/blog/introducing-red-hat-openshift-4.10 |
 | Containers | [Red Hat](https://partners.amazonaws.com/partners/001E000000Rl0xiIAB) | [ROSA HCP](https://aws.amazon.com/rosa/) | Release: https://docs.redhat.com/en/documentation/red_hat_openshift_service_on_aws/4/html-single/introduction_to_rosa/index#rosa-sdpolicy-aws-instance-types-graviton_rosa-hcp-instance-types |
 | Data Science & ML | [Anaconda](https://partners.amazonaws.com/partners/0010L00001qFzbyQAC) | [Anaconda](https://www.anaconda.com/products/commercial-edition) | Release: https://docs.anaconda.com/anaconda/install/graviton2/, https://repo.anaconda.com/pkgs/main/linux-aarch64/<br>Announcement: https://www.anaconda.com/blog/anaconda-aws-graviton2 |
+| Database | [Aerospike](https://aerospike.com/) | [Aerospike Database](https://aerospike.com/products/database/) |  Release: [Aerospike Showcases New Levels of Database Performance, Price, and Sustainability on Graviton3 at AWS re:Invent](https://aerospike.com/news/press-release/announcing-aerospike-on-graviton3/) <br>INTERNAL TFC: [Aerospike on AWS best practices](https://w.amazon.com/bin/view/AWS/Teams/TFC/AMT/Resources/AerospikeonAWS) |
```

Comment:
Please remove amazon internal only link.

Reply:
@JeffUnderhill , can you please remove the internal link as suggested above?